### PR TITLE
[GSoD 2021] Added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by iamrajiv**
Related: wechaty/docusaurus#1012

Screenshot 2021-07-06 at 9 48 28 PM
Screenshot 2021-07-06 at 9 48 37 PM

### CODE REVIEW

**Comment by iamrajiv** [comment]
@proudofsimin @Rohitesh-Kumar-Jain Please take a look.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Guidelines for Contributing
+
+Thank you for your time on Wechaty OpenAPI.
+
+We have [gRPC](https://github.com/wechaty/grpc) for [Wechaty Puppet](https://github.com/wechaty/wechaty-puppet), and we want a RESTful API as well, so we built [OpenAPI Specification](https://www.openapis.org/) (OAS, former [Swagger](https://swagger.io/)) on top of gRPC, with the power of [gRPC OAS Gateway](https://github.com/grpc-ecosystem/grpc-gateway).
+
+## How to [contribute](https://github.com/wechaty/openapi/blob/master/CONTRIBUTING.md)
+
+- Write a blog about Wechaty OpenAPI.
+- Create a tutorial for Wechaty OpenAPI.
```
Comment (proudofsimin) [inline CONTRIBUTING.md:10]: How do you think if we add a link to [General Wechaty blog post guidelines](https://github.com/wechaty/wechaty.js.org)?

**Review by proudofsimin — COMMENTED** [review]
This looks so nice!

File: CONTRIBUTING.md (hunk 1 above)
Comment (iamrajiv) [reply CONTRIBUTING.md:10]: @proudofsimin Added the link.

**Review by proudofsimin — APPROVED** [review]

**Comment by lijiarui** [comment]
Thanks for improving this!

I have a suggestion to combine all of the contributing guides together to our docs, see: https://github.com/wechaty/wechaty.js.org/issues/1111

**Comment by iamrajiv** [comment]
@lijiarui Great! so repo won't have CONTRIBUTING.md file?

**Review by huan — CHANGES_REQUESTED** [review]
Please link to our official docs website at https://wechaty.js.org/docs/contributing/

**Comment by iamrajiv** [comment]
@huan I have updated to contributing file of this PR and added the https://wechaty.js.org/docs/contributing/ link. I have made PR for improvement of contributing  guidelines here wechaty/wechaty.js.org#1276, please review

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
It's a great CONTRIBUTING file, thank you very much!

**Pull request merged**